recipes/unpaper/yum_requirements.txt

```diff
@@ -0,0 +1,8 @@
+docbook-dtds
```

Comment:
Is there a reason to not add these packages to conda-forge so they can be explicit dependencies?

Reply:
Laziness mostly :-) because they are not really needed. I did this originally because I was getting xslt errors during the build and https://wiki.archlinux.org/index.php/DocBook#Compilation_errors suggested that installing docbook as a solution. As it turns out, all I really needed was libxml2, which I have now used instead. Thanks for catching this!